reddit_liveupdate/media_embeds.py

```diff
@@ -129,6 +145,41 @@ def for_url(cls, url, autoplay=False, maxwidth=_EMBED_WIDTH):
 
         return _EmbedlyCardFallbackScraper(url, scraper)
 
+# mostly lifted from the EmbedlyScraper in r2
+class _LiveEmbedlyScraper(_OEmbedScraper):
+    OEMBED_ENDPOINT = "https://api.embed.ly/1/oembed"
+
+    @classmethod
+    def matches(cls, url):
+        return True
```

Comment:
Yeah, should attempt to scrape any url for the mobile embeds. If there are some cases that we don't want to handle could add some conditions here but I don't know of any right now.